.changeset/rude-bees-deliver.md

```diff
@@ -0,0 +1,5 @@
+---
+'@craftjs/core': minor
```

Comment:
Should be `patch` 

https://github.com/prevwong/craft.js/blob/main/CONTRIBUTING.md#pull-requests